modules/ecs6-class/point.js

```diff
@@ -1,14 +1,23 @@
 class Point {
     constructor({x=0, y=0}={}) {
+        if(typeof(x)!="number" || typeof(y)!="number"){
+            throw new Error('argument must be type number')
```

Comment:
which parameter is not a number?
use the three equals sign

Reply:
if(typeof(x)==="number" || typeof(y)==="number"){